pkgs/by-name/ap/apbs/package.nix

```diff
@@ -95,12 +92,9 @@ stdenv.mkDerivation (finalAttrs: {
     "-DAPBS_LIBS=mc;maloc"
     "-DCMAKE_MODULE_PATH=${fetk}/share/fetk/cmake;"
     "-DENABLE_TESTS=1"
+    "-DCMAKE_C_FLAGS=-Wno-error=incompatible-pointer-types"
   ];
 
-  env = lib.optionalAttrs stdenv.cc.isClang {
-    NIX_CFLAGS_COMPILE = "-Wno-error=incompatible-function-pointer-types";
-  };
-
```

Comment:
Note that these are not the same compiler flags. `incompatible-function-pointer-types` does not work with gcc. Does `incompatible-pointer-types` do the same with Clang?

Reply:
I have no idea honestly what Clang does. GCC throws this error and does not accept the "fuction-" version.

Reply:
The current version also works perfectly fine with `llvmPackages.stdenv` :sheep: :tada: